lib/puloon/puloonrs232.js

```diff
@@ -154,7 +154,10 @@ PuloonRs232.prototype._process = function _process (data) {
     if (data[0] === EOT) {
       this.setState('idle')
       this.emit('response', this.response)
-      if (this.responseCallback) this.responseCallback(null, this.response)
+      if (this.responseCallback) {
+        if (_.has('err', this.response)) this.responseCallback(this.response)
```

Comment:
In this way this will cause the promise [here](https://github.com/lamassu/lamassu-machine/blob/9eb060500af2c998caf1b161f7ae6b5fa04ace62/lib/puloon/puloonrs232.js#L202) to be reject and catcher [here](https://github.com/lamassu/lamassu-machine/blob/9eb060500af2c998caf1b161f7ae6b5fa04ace62/lib/puloon/puloonrs232.js#L254)